NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -68,6 +70,63 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
                 }
             );
 
+            Field<NonNullGraphType<ListGraphType<NonNullGraphType<TransferNCGHistoryType>>>>(
+                "transferNCGHistories",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<ByteStringType>>
+                    {
+                        Name = "blockHash"
+                    },
+                    new QueryArgument<AddressType>
+                    {
+                        Name = "recipient"
+                    }
+                ), resolve: context =>
+                {
+                    BlockHash blockHash = new BlockHash(context.GetArgument<byte[]>("blockHash"));
+
+                    if (!(standaloneContext.Store is { } store))
+                    {
+                        throw new InvalidOperationException();
+                    }
+
+                    if (!(store.GetBlock<NCAction>(blockHash) is { } block))
+                    {
+                        throw new ArgumentException("blockHash");
+                    }
+
+                    var recipient = context.GetArgument<Address?>("recipient");
+
+                    var txs = block.Transactions.Where(tx =>
+                        tx.Actions.Count == 1 &&
+                        tx.Actions.First().InnerAction is TransferAsset transferAsset &&
+                        (!recipient.HasValue || transferAsset.Recipient == recipient) &&
+                        transferAsset.Amount.Currency.Ticker == "NCG" &&
+                        store.GetTxExecution(blockHash, tx.Id) is TxSuccess);
+
+                    TransferNCGHistory ToTransferNCGHistory(TxSuccess txSuccess)
+                    {
+                        var rawTransferNcgHistories = txSuccess.FungibleAssetsDelta.Select(pair =>
+                                (pair.Key, pair.Value.Values.First(fav => fav.Currency.Ticker == "NCG")))
```

Comment:
IMO, we may need to store currency for NCG to `StandaloneContext`... (even later)